guides/models.md

```diff
@@ -77,4 +80,20 @@ class Customer < ActiveRecord::Base
     # some calculation
   end
 end
+
+# app/models/concerns/commentable
+module Commentable
+  # Concern should be named with `able` prefix
+  extend ActiveSupport::Concern
+  # EXTENDing is required
```

Comment:
Probably we can omit this comment because this is known from the official Rails guides. You may add a link to concerns guide page somewhere instead.